docs/ROADMAP.md

```diff
@@ -2,39 +2,34 @@
 
 ## About this document
 
-This document should serve as a reference point for WasmEdge users and contributors to understand where the project is heading, and help determine if a contribution could be conflicting with a longer term plan.
+This document serves as a reference point for WasmEdge users and contributors to understand where the project is heading, and help to determine if a contribution could be conflicting with a longer term plan. Additionally, this document also describes the rules and steps of proposing and retiring roadmap entries.
 
-## How to help?
+## Planning for Roadmap
 
-Discussion on the roadmap can take place in threads under [Issues](https://github.com/WasmEdge/WasmEdge/issues). Please open and comment on an issue if you want to give suggestions and feedback to items in the roadmap. Please review the roadmap to avoid potential duplicated efforts.
+WasmEdge roadmap will be updated quarterly with the following steps.
 
-## How to add an item to the roadmap?
-
-Please open an issue to track any initiative on the roadmap of WasmEdge (Usually driven by new feature requests). We will work with and rely on our community to focus our efforts to improve WasmEdge.
+1. WasmEdge maintainers will raising an issue on the upcoming roadmap at least 14 days before the start of a new quarter.
+2. Any individual can request inclusion in the roadmap by commenting on the issue and volunteering as an assignee.
+3. WasmEdge maintainers will collect the proposed roadmap entries along with their respective timelines and assignees, then update them within the issue.
+4. Existing roadmap entries which not reach their deadlines will be retained in the new quarter.
+5. Roadmap entries that have reached their deadlines will be marked as completed or dropped, and the associated issues will be closed.
```

Comment:
Complete, drop, or update deadline? Since maybe the entry is delayed by accident.

Reply:
We should figure out if there is any GitHub-Bot that can help to auto-close the stale proposals.

Reply:
Sure, I think the deadline can be updated due to some reason. This can be discussed in the new quarter roadmap.

Reply:
The timeslot is based on quarters now. How about:
1. Propose a quarter -> Stale
2. Propose two quarters -> Close
It should be fine to drop a proposal if we have no plan to implement it in half a year.

Reply:
Sounds reasonable.

Reply:
Updated